site/_includes/header.html

```diff
@@ -98,7 +98,7 @@
                       <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Documentation</a>
                       <ul class="dropdown-menu">
                         <li><a href="docs/home">Documentation Home</a></li>
-                        <li><a href="{{ site.projurl }}/docs/javadoc/index.html">Javadoc</a></li>
```

Comment:
Thank you @dlaboss for review and good pointing.

- I agree using properties is much clearer rather than hard coding value. I will replace javadoc-related-url all with using properties in next day.
- We can change ```Quarks``` to ```{{ site.projname }}``` but it has been used too many times. I worried about it may cause readability reduction. But if you mean "Nested Property"(I'm not sure how I can say in english.. It something like ```projname:quarks``` -> ```url: http://%{projname}.incubator.apache.org``` in define properties phase) , then it looks very helpful